exosphere-shared/src/docker-compose.ls

```diff
@@ -0,0 +1,59 @@
+require! {
+  \observable-process : ObservableProcess
+}
+
+
+class DockerCompose
+
+  @build-all-images = ({write}, done) ->
+    new ObservableProcess('docker-compose build'
+                          stdout: {write}
+                          stderr: {write})
+      ..on 'ended', done
+
+  @pull-all-images = ({write}, done) ->
+    new ObservableProcess('docker-compose pull'
+                          stdout: {write}
+                          stderr: {write})
+      ..on 'ended', done
+
+
+  @run-all-images = ({env, write}, done) ->
+    new ObservableProcess('docker-compose up'
+                          env: env
+                          stdout: {write}
+                          stderr: {write})
+      ..on 'ended', done
+
+
+  @kill-container = ({service-name, write}, done) ->
+    new ObservableProcess("docker-compose kill #{service-name}"
```

Comment:
Let's follow the lessons learned from the DockerHelper and provide the arguments as an array, so that we don't run into issues with escaping strings for shell usage.

```ls
new ObservableProcess(['docker-compose', 'kill', service-name]